src/Daemon/BaseDaemon.cpp

```diff
@@ -267,6 +267,19 @@ void BaseDaemon::initialize(Application & self)
 
     ConfigProcessor(config_path).savePreprocessedConfig(loaded_config, "");
 
+#if USE_SSL
+    // Decrypt encrypted elements in config
```

Comment:
The statement in l. 272 is not clear for the reader. Needs better explanation, maybe include a XML example as code comment.